source/elements/oneDAL/source/algorithms/index.rst

```diff
@@ -1,14 +1,8 @@
-============
- Algorithms
-============
+==========
+Algorithms
+==========
 
 .. toctree::
```

Comment:
I'd suggest you use here a maxdepth limit for toc, otherwise you get a table of content that is too messy

```
.. toctree::
   :maxdepth: 2
```
![toc-2max](https://user-images.githubusercontent.com/17865025/76803668-6a807380-67eb-11ea-8b76-aff0f110edd6.PNG)
![toc-nolimit](https://user-images.githubusercontent.com/17865025/76803669-6bb1a080-67eb-11ea-8cb4-4da6b6c31825.PNG)

Reply:
Fixed by c7866e3